typescript/packages/well-log-viewer/src/SyncLogViewer.tsx

```diff
@@ -64,7 +64,7 @@ export interface SyncLogViewerProps {
     /**
      * Object from JSON file describing single well log data.
      */
-    welllogs: WellLog[];
+    welllogs: (WellLogSet | WellLogCollection)[];
```

Comment:
Suggest we document the behaviour according to the different types allowed.

Reply:
Slightly unsure what you mean, but I tweaked the wording on this (and `WellLogViewProps`) slightly; is the wording better now?